pkg/controller/plan/kubevirt.go

```diff
@@ -784,24 +793,28 @@ func (r *KubeVirt) EnsureGuestConversionPod(vm *plan.VMStatus, vmCr *VirtualMach
 		err = liberr.Wrap(err)
 		return
 	}
-	pod := &core.Pod{}
-	if len(list.Items) == 0 {
-		pod = newPod
-		err = r.Destination.Client.Create(context.TODO(), pod)
-		if err != nil {
-			err = liberr.Wrap(err)
+
+	for _, pod := range list.Items {
+		if !strings.Contains(pod.Name, "pvcinit") {
+			// There is a conversion pod.
```

Comment:
as I wrote you offline, I think it would be better to set these pods with a label that reflects their purpose rather than having this logic - can we do that separately?